controllers/managedocs_controller.go

```diff
@@ -911,12 +911,18 @@ func (r *ManagedOCSReconciler) reconcileAlertRelabelConfigSecret() error {
 		}
 
 		alertRelabelConfig := []struct {
-			TargetLabel string `yaml:"target_label,omitempty"`
-			Replacement string `yaml:"replacement,omitempty"`
-		}{{
-			TargetLabel: "namespace",
-			Replacement: r.namespace,
-		}}
+			SourceLabels []string `yaml:"source_labels"`
```

Comment:
```suggestion
			SourceLabels []string `yaml:"source_labels,omitempty"`
```

Reply:
Adding `omitreply` was not working, tests were failing and even the deployment too.